eclair-core/src/main/resources/reference.conf

```diff
@@ -109,6 +109,7 @@ eclair {
   payment-request-expiry = 1 hour // default expiry for payment requests generated by this node
   multi-part-payment-expiry = 60 seconds // default expiry for receiving all parts of a multi-part payment
   min-funding-satoshis = 100000
+  max-funding-satoshis = 100000000 // 1BTC
```

Comment:
The idea was to have wumbo as first limit and another max-funding-satoshis with a relatively high limit in case wumbo is enabled. But i like this solution too, we default to the wumbo limit and force the user to increase the max funding accordingly, also the default is also used in the test now (as of a65c090) .